### PR TITLE
Updated Mongod to 2.6.3, updated mongo C# Drivers.

### PR DESCRIPTION
**Pull request by lawrencegripper**

### CODE REVIEW

**Comment by kevfromireland** [comment]
Any chance of getting this merged and a new nuget package published?

File: MongoDb.Embedded.sln [hunk 1]
```diff
@@ -8,10 +10,22 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{80BA23A4-6ADE-4986-8F87-5CF5A87E9055}"
 	ProjectSection(SolutionItems) = preProject
 		.nuget\NuGet.Config = .nuget\NuGet.Config
-		.nuget\NuGet.exe = .nuget\NuGet.exe
 		.nuget\NuGet.targets = .nuget\NuGet.targets
 	EndProjectSection
 EndProject
+Project("{911E67C6-3D85-4FCE-B560-20A9C3E3FF48}") = "NuGet", ".nuget\NuGet.exe", "{2A270022-EEC9-427D-8F98-420C594F6E51}"
```
Comment (bytenik) [inline MongoDb.Embedded.sln]: Unfortunately all of this included NuGet still needs to come out.

**Comment by bytenik** [comment]
I can but at this point do you want to update for the 2.0 driver?

**Comment by lawrencegripper** [comment]
Hey, will try and get round to fixing and resubmitting the PR without nuget dependencies at some point. In the meantime I released the forked version to nuget a while back for my project, can get it here. https://www.nuget.org/packages/MongoDB.Embedded.UpdatedFork/

**Comment by kevfromireland** [comment]
@bytenik : Yeah, I was really just checking if this is still alive. I'm happy to send a PR your way that updates `mongod` and moves to newest driver if that sounds good?

**Comment by bytenik** [comment]
Sounds good.

**Comment by kevfromireland** [comment]
With #2 from @spencerthang being merged, I think this PR can be closed now?

Would you be able to publish a new NuGet package as well?

**Pull request closed without merging**